docs/resource-schemas.md

```diff
@@ -45,6 +45,11 @@ assets:
       - data-display
     framework: react
     platform: web
+    docs:
+      usagePath: 'https://github.com/carbon-design-system/carbon-website/blob/main/src/pages/components/accordion/usage.mdx'
```

Comment:
`docs` makes sense as the key. Other options: `documentation`, `pages`... since we can always add `navData` to assets later on if we need to add freeform page support to assets, I think this still makes sense to call these docs. It's asset documentation.

Also, I like how the docs keys end in `Path`, as that's consistent with the asset `thumbnailPath` and the library `packageJsonPath`.